packages/bundler-plugin-core/src/index.ts

```diff
@@ -333,6 +345,16 @@ function generateGlobalInjectorCode({ release }: { release: string }) {
             {};
 
     _global.SENTRY_RELEASE={id:"${release}"};`;
+
+  if (project) {
+    const key = org ? `${project}@${org}` : project;
+    code += `
+      _global.SENTRY_RELEASES=_global.SENTRY_RELEASES || {};
+      _global.SENTRY_RELEASES["${key}"]={id:"${release}"};
```

Comment:
l: can we make this hidden behind a flag? I'd rather not inject things unnecessarily. I recognize this is a breaking change, but given this is undocumented I'm fine making the change.

Reply:
I think that's fair and it would not bloat the bundle unnecessarily for all 99% of users. Will do.

Reply:
I agree, is this something that people could have to opt-in to? If that's easy to do, I would say let people opt in (then, in a hypothetical world where we have a better understanding of which options are set by users, we can take better decisions in the future about this functionality). Seems like it is something pretty specific to want.

Reply:
[a3ca32f](https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/102/commits/a3ca32fbb9ffc185e91940735469fbd58de768e4)